EPM VSAT documentation (#924)
SkylineCommunications/dataminer-docs: user-guide/Standard_Apps/EPM/EPM_VSAT/VSAT_components.md

> [!TIP]
> For more information, see [Verizon iDirect Evolution Platform Collector](https://catalog.dataminer.services/result/driver/5827).

#### iDirect Dialog Platform VSAT

This is the collector connector required for the iDirect (formerly Newtec) Dialog Platform with Time Series Database (TSDB) integration.

> [!TIP]
> For more information, see [Newtec Dialog Platform VSAT](https://catalog.dataminer.services/result/driver/7342).

#### CMDB

This is the collector connector that helps form relationships between various collectors and technology that are traditionally unrelated. It can provide additional business details of the customer to enhance the metrics and KPIs needed that are not obtained by the NMS collectors alone.

### EPM connectors

EPM connectors constitute the core of the EPM Solution. They carry out the necessary logic to aggregate and display the information of the different topology levels. EPM connectors rely on collectors and system connectors to feed information into the EPM engine.

EPM connectors ship with the EPM Solution package.

## Peripherals

Peripherals are optional but not required as part of a standard deployment.

### Sun outage

> DataMiner predicts, detects falls, and offers the current state details of each remote in the VSAT system referring to the sun outage events that might affect the transmission. These events normally occur twice a year when the day arc takes place directly behind the line of sight of a satellite and an earth station.

Reviewer: This first sentence isn't entirely clear to me. What exactly is it that DataMiner predicts and detects?

### Weather

> DataMiner has flexibility of integration with any weather forecast service as it considers this variable a key factor capable of degrading and temporarily interrupting the signal delivery via VSAT. Operators receive automated detailed descriptions of the physical and remote falls that were correlated to weather conditions.

Reviewer: What do you mean with falls here and in the description for PLM below? I'm not sure if this is clear enough.

### PLM

During planned maintenance (PLM), DataMiner suppresses any diagnostic and platform falls alerts related to the event process. It works in conjunction with SRA modules.

## Correlation rules

The VSAT EPM Solution uses the following Correlation rules:

> - **VerSatOnReportCorrelation**: Correlation rule in charge of the synchronization of RDS subscribers (KPI, Entry, DCAT, etc.) by monitoring for **OnChange* and *OnUpdate* information events.

Reviewer: ```suggestion
- **VerSatOnReportCorrelation**: Correlation rule in charge of the synchronization of RDS subscribers (KPI, Entry, DCAT, etc.) by monitoring for *OnChange* and *OnUpdate* information events.
```

> - **VrzVsatEventManager**: Correlation rule in charge of picking up all the events/alarms that should create a ticket request entry in the WM Ticketing protocol and send it to the correlation that send the requests to the proper ticketing element.

Reviewer: ```suggestion
- **VrzVsatEventManager**: Correlation rule in charge of picking up all the events/alarms that should create a ticket request entry in the WM Ticketing protocol and send it to the Correlation rule that sends the requests to the proper ticketing element.
```

- **VerSatOnSunOutageAutomation**: Automation script in charge of handling sun outage protocol events.

  This Automation script is dependent on the following protocols:

  - Verizon WM DSM: This system protocol allows the Verizon DSM SO protocol to communicate with the Profile Manager.

  - Verizon DSM SO: This protocol is used to gather information via inter-element communication that will be exported to a location used by the Generic Sun Outage protocol. The information gathered consists of key parameters used during the calculation of sun outages. This protocol is purely a system protocol with this sole responsibility.

  The *VerSatOnSunOutageAutomation* script oversees collecting data from an information event triggered by Verizon DSM SO and passes it to Verizon WM DSM in order for that workflow manager to create, update, or remove a Profile Manager object. The information event is triggered from the context menus on the DSM SO protocol.

- **EPM BE Message Handler**: Automation script in charge of handling EPM back-end messages.

  This Automation script is dependent on the following protocols:

>   - The [collector](#collector-connectors) that triggered the initial EPM Message handler script:

Reviewer: Capitalization is inconsistent here and in line 103 and 115: Either we should fully capitalize "EPM Message Handler" script to refer to its exact name, or we should not capitalize this at all if it's intended as a description instead of the exact name.

    - iDirect Evolution

    - iDirect (Newtec) Dialog Platform VSAT

>   - Verizon VSAT Platform Manager: This CPE protocol allows the aggregation of KPIs from different collector elements deployed in the Verizon infrastructure (e.g. Verizon iDirect Evolution Platform Collector).

Reviewer: ```suggestion
  - Verizon VSAT Platform Manager: This EPM protocol allows the aggregation of KPIs from different collector elements deployed in the Verizon infrastructure (e.g. Verizon iDirect Evolution Platform Collector).
```

The term "CPE" is the outdated equivalent of "EPM" and should be avoided if at all possible.

>   The *EPM BE Messange Handler* script is in charge of communicating to the collector that the ID assignment files from the VSAT Platform Manager are ready for ingestion after the EPM backends are done processing and setting the topology data in their respective tables.

Reviewer: ```suggestion
  The *EPM BE Message Handler* script is in charge of communicating to the collector that the ID assignment files from the VSAT Platform Manager are ready for ingestion after the EPM back ends are done processing and setting the topology data in their respective tables.
```

Ideally, "back end" should be spelled with a space in between when used as a noun.

- **EPM FE to BE**: Operates within the messaging system domain taking care of communication about the collector and entity the processed data originated from, between EPM front-end elements and the back-end elements.

  This Automation script is dependent on the following protocols:

  - The [collector](#collector-connectors) that triggered the initial EPM Message handler script:

    - iDirect Evolution

    - iDirect (Newtec) Dialog Platform VSAT

>   - Verizon VSAT Platform Manager: This CPE protocol allows the aggregation of KPIs from different collector elements deployed in the Verizon infrastructure (e.g. Verizon iDirect Evolution Platform Collector).

Reviewer: ```suggestion
  - Verizon VSAT Platform Manager: This EPM protocol allows the aggregation of KPIs from different collector elements deployed in the Verizon infrastructure (e.g. Verizon iDirect Evolution Platform Collector).
```

- **EPM Message Handler**: Automation script in charge of handling EPM messages.

  This Automation script is dependent on the following protocols:

  - The [collector](#collector-connectors) that triggered the initial EPM Message handler script:

    - iDirect Evolution

    - iDirect (Newtec) Dialog Platform VSAT

>   - Verizon VSAT Platform Manager: This CPE protocol allows the aggregation of KPIs from different collector elements deployed in the Verizon infrastructure (e.g. Verizon iDirect Evolution Platform Collector).

Reviewer: ```suggestion
  - Verizon VSAT Platform Manager: This EPM protocol allows the aggregation of KPIs from different collector elements deployed in the Verizon infrastructure (e.g. Verizon iDirect Evolution Platform Collector).
```

>   The *EPM Message Handler* script is triggered from the collector after exporting the ID request files for the entities. It communicates to the EPM frontend to ingest the request files.

Reviewer: ```suggestion
  The *EPM Message Handler* script is triggered from the collector after the ID request files for the entities are exported. It tells the EPM front end to ingest the request files.
```

Like "back end", "front end" should be written as two words when it's a noun.
